LinkedIn-Certifications-Manager/README.md

```diff
@@ -0,0 +1,24 @@
+# LinkedIn Certification Manager
+
+A CLI based tool to automatically add certification to your LinkedIn profile. 
+
+## How to use
+- Make sure you have Google Chrome browser.
+- Download [chromedriver](https://chromedriver.chromium.org/)  for your version of Chrome and move it to LinkedIn-Certification-Manager directory  
```

Comment:
Instead of this manual task, use web_driver_manager library which will install the driver on its own. Refer to the implementation here: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/797/files